docs/cheaha/open_ondemand/ood_jupyter.md

```diff
@@ -20,11 +23,20 @@ module load cuDNN/8.9.2.26-CUDA-12.2.0
 
 For information on which versions of CUDA to load for Tensorflow and PyTorch, please see [Tensorflow Compatibility](../slurm/gpu.md#tensorflow-compatibility) and [PyTorch Compatibility](../slurm/gpu.md#pytorch-compatibility).
 
+For information on partition and GPU selection, please review our [hardware information page](../hardware.md)
+
```

Comment:
You may also want to link to our GPU page: https://docs.rc.uab.edu/cheaha/slurm/gpu/